setup.py

```diff
@@ -69,10 +71,10 @@
     'arrow>=0.13.0',
     'Flask-BabelEx>=0.9.3',
     'invenio-jsonschemas>=1.0.0,<1.1.0',
-    'invenio-records-rest>=1.5.0,<1.6.0',
+    'invenio-records-rest>=1.6.0,<1.7.0',
     'invenio-records>=1.3.0,<1.4.0',
-    'invenio-records-files>=1.1.1,<1.2.0',
-    'invenio-records-permissions>=1.0.0a3',
+    'invenio-records-files>=1.2.0,<1.3.0',
+    'invenio-records-permissions>=1.0.0a5',
```

Comment:
```suggestion
    'invenio-records-permissions>=1.0.0a4',
```

I will change and merge.